rpm/dnsperf.spec

```diff
@@ -8,7 +8,7 @@ License:        Apache-2.0
 URL:            https://www.dns-oarc.net/tools/dnsperf
 # Source needs to be generated by dist-tools/create-source-packages, see
 # https://github.com/jelu/dist-tools
-Source0:        %{name}_%{version}.orig.tar.gz
+Source0:        https://www.dns-oarc.net/files/dnsperf/%{name}-%{version}.tar.gz?/%{name}_%{version}.orig.tar.gz
```

Comment:
I will not be able to use this because the same SPEC-file is used to automatically create packages from develop branch for pre-releases and they are not tagged.

Reply:
Sorry, should have tested before :) this actually works, nice!